### DISCUSSION TITLE
Priority list & Log in console

### DISCUSSION THREAD

**Discussion by egorcreep** [RakambdaOrg/ChannelPointsMiner#237] [Q&A] [answered]
Hello, 

I just downloaded your project instead of one from Tkd-Alex, since it started to work unstable. Your project is working perfect, so I want to ask a few thing to get it worked like a previous one.

So I got a questions,

The first one is it possible to make a streamers priority list , like this:
"streamer1",
"streamer2",
"streamer3", 
"streamer4",
"streamer5",
etc..
so if streamer 2,4,5 is online, script would watch streamer  2,4..

And the second one is it possible to make logs in console without unnecessary messages?
←[32m00:32:01,166 INFO [pool-3-thread-15] f.r.c.m.l.LoggerEventListener {account_name=username, streamer_name=streamer, websocket_id=, websocket_topic=community-points-user-v1.37465515} - Points earned [+60 | CLAIM | 53.32K]←[m
Is it possible to make it simple like "streamer_name Points earned [+60 | CLAIM | 53.32K]"
unnecessary messages like:  Parsed message: MessageResponse, Received Websocket message, Sending WebSocket message,  Updating spade URL, Updating ban status, Updating channel points context, etc..
So basically I just want to receive logs only regarding my points changing.

I am just an ordinary user who wants to use a great project but do it with simplicity and convenience ;)

Thank you, I wish you good luck with your project.

**Comment by Rakambda** [comment, marked as answer]
Hello,

For the logs, did you already change them to have them at a INFO level? That should clean them a little but still show "unwanted info".
As it is it won't be possible to have them the way it is done in Tkd. This is just my personal take on it as for me logs are meant to understand what happened in case of an issue, not really display pretty messages for the user.
For that purpose there is the discord integration that'll do it the same way as Tkd's. If discord is not an option I'm open to other integrations if people use them (I know on Tkd's there's an integration with Telegram for example).

For the priorities that's possible. The way this works is that each x amount of time a score is computed for each streamer based on the conf you put in the "priorities" section, the 2 best scores will be selected. 
If you want a pre-defined order you can give them a constant score (like streamer1 has a score of 100, streamer2 has a score of 99,... This way they'll be mined in order). Or you can use the index in the config (this is used when streamers have the same score).

---

I know the readme is quite bulky and not fun to navigate into but there's a section that may give more details: Settings > Streamer Settings > Priorities.

You also can have some answers in #236
